omod/src/main/java/org/openmrs/module/aijar/web/controller/AppointmentController.java

```diff
@@ -0,0 +1,54 @@
+
+package org.openmrs.module.aijar.web.controller;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+ @Controller
+//@RequestMapping("/rest/ugandaemr/appointments") 
+public class AppointmentController { 
+
+    AdministrationService administrationService = Context.getAdministrationService();
+
+    @RequestMapping(value = "/rest/ugandaemr/appointments", method = RequestMethod.GET)
```

Comment:
@reagan the value has to be RestConstants.VERSION_1 + "/ugandaemr/appointmentcount"